docs/persistence.rst

```diff
@@ -0,0 +1,12 @@
+Persistence
+-----------
+
+By default, you do not need to provide a :code:`database_path` to the :code:`AsynchronousLogstashHandler`.
+There are a couple of things you should keep in mind should you choose to go down this path.
```

Comment:
I'm not a native speaker but I think replacing "should" by "if" would make it more readable.